ios/SecuritySdk/Classes/Digital Signature/Secure Enclave/SecureEnclaveDigitalSignatureKeyManager.swift

```diff
@@ -21,10 +22,35 @@ public class SecureEnclaveDigitalSignatureKeyManager: DigitalSignatureKeyManager
     public init(tag: String) {
         self.tag = tag
     }
+    
+    // MARK: - Public Methods (DigitalSignatureSigner)
 
-    // MARK: - Public Methods (DigitalSignatureKeyManager)
+    public func sign(with data: Data) throws -> Data {
```

Comment:
so I don't know anything about swift, but is there a way to scope this to 'just trifle' and not let trifle clients actually call this method directly (and same with similar ones) - naively 'internal' looks to be the way based on other code here?

Reply:
its not possible to reduce scope if the protocol is declared as public